src/index.ts

```diff
@@ -10,6 +10,7 @@ export { createOptionFromJSON } from './fp-ts/createOptionFromJSON'
 export { createOptionFromNullable } from './fp-ts/createOptionFromNullable'
 export { createSetFromArray } from './fp-ts/createSetFromArray'
 export { createStrMapFromDictionary } from './fp-ts/createStrMapFromDictionary'
+export { fromNullable } from './fp-ts/fromNullable'
```

Comment:
Not sure why this is in the `fp-ts` folder, it doesn't look strictly related. I think that we can move it to the root folder